### PR TITLE
Arturotepaleaguilar/ch33/cotroller for the posts

### PR DESCRIPTION
**Pull request by OleksandrVakhno**

### CODE REVIEW

File: ProjectsNetwork/Controllers/PostsController.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            project.CreationDate = DateTime.Now;
+            ClaimsPrincipal currentUser = this.User;
+            var currentUserID = currentUser.FindFirst(ClaimTypes.NameIdentifier).Value;
+            project.UserId = currentUserID;
+
+            //return Json(project);
+
+            this._projectRepository.Insert(project);
+            this._projectRepository.Save();
+            var projects = this._projectRepository.GetAll();
```
Comment (OleksandrVakhno) [inline ProjectsNetwork/Controllers/PostsController.cs]: Probably should include error checking. What if no user is found? What if the project is null? What if save throws an exception for some reason?

**Review by OleksandrVakhno — COMMENTED** [review]
Otherwise looks decent.

File: ProjectsNetwork/Controllers/PostsController.cs (hunk 1 above)
Comment (atepag) [reply ProjectsNetwork/Controllers/PostsController.cs]: got it, will look over that

**Review by atepag — APPROVED** [review]
This should avoid basic errors.

**Pull request merged**